### PR TITLE
Update tagline, description, what is, and goals

### PR DESCRIPTION
**Pull request by mraible**
I think this might look better if we don't have the full "what is JHipster" in the initial paragraph because there are five lines of text in the paragraph on a desktop monitor. Here's what it looks like if we include the full text.

full-text

And here's what it looks like with abbreviated text that fits on one line.

abbreviated-text

Below is a screenshot of the What is JHipster section.

what-is-jhipster

### CODE REVIEW

**Comment by atomfrede** [comment]
I agree. Having the short tagline on the initial paragraph is better.

**Comment by SudharakaP** [comment]
I vote for the sort tagline as well. :smile:

**Comment by mraible** [comment]
I added another goal to the list:

* Infrastructure as code so you can quickly deploy to the ☁️

I also re-worked the samples list so it's easier to read.

samples

File: index.html [hunk 1]
```diff
@@ -26,16 +26,19 @@
                     <div class="intro-text">
                         <span class="brand">Greetings, Java Hipster!</span>
                         <hr class="star-light">
-                        <span class="open-collective-info">
-                            JHipster is now on <a href="https://opencollective.com/generator-jhipster" target="_blank" rel="noopener">Open Collective</a>,
+                        <p class="project-description">
+                            JHipster is a development platform to quickly generate, develop, & deploy modern web applications & microservice architectures.
```
Comment (jdubois) [inline index.html:30]: "Web" instead of "web"?

File: index.html [hunk 2]
```diff
@@ -482,24 +485,22 @@ <h2 class="heading">What is JHipster?</h2>
...
-                        JHipster is a development platform to generate, develop and deploy
-                        <a class="dark" href="http://projects.spring.io/spring-boot/" target="_blank" rel="noopener">Spring Boot</a> + <a class="dark" href="https://angular.io/" target="_blank" rel="noopener">Angular</a> / <a class="dark" href="https://reactjs.org/" target="_blank" rel="noopener">React</a> / <a class="dark" href="https://vuejs.org/" target="_blank" rel="noopener">Vue</a>
-                        Web applications and Spring microservices. <br>
+                    <p class="lead font-weight-normal">
+                        JHipster is a development platform to quickly generate, develop, and deploy modern web applications and microservice architectures. We support many frontend technologies, including Angular, React, and Vue. We even have mobile app support for Ionic and React Native! On the backend, we support Spring Boot (with Java or Kotlin), Micronaut, Quarkus, Node.js, and .NET. For deployment, we embrace cloud native principles with Docker and Kubernetes. Deployment support exists for AWS, Azure, Cloud Foundry, Google Cloud Platform, Heroku, and OpenShift.
```
Comment (jdubois) [inline index.html:489]: Same here, "Web" instead of "web"?

File: _layouts/default.html [hunk 3]
```diff
@@ -4,12 +4,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">{% if '/index.html' == page.url or '/' == page.url %}
-    <meta name="description" content="JHipster is a fully Open Source, widely used application generator. Easily create high-quality Spring Boot + Angular/React projects!">
+    <meta name="description" content="JHipster is an open source, widely used application generator, made for modern full-stack developers.">
```
Comment (jdubois) [inline _layouts/default.html:7]: "open source" or "Open Source"?

File: _layouts/default.html [hunk 4]
```diff
@@ -4,12 +4,12 @@
...
     <meta name="author" content="JHipster Team">
     <meta name="twitter:card" content="summary_large_image" />
     <meta name="twitter:site" content="@jhipster" />
     <meta name="twitter:title" content="Generate a full stack Java + Angular/React application in a few steps" />
-    <meta name="twitter:description" content="JHipster is a fully Open Source, widely used application generator. It leverages cool technologies like Spring Boot, Angular and React to give you a high level of productivity and quality." />
+    <meta name="twitter:description" content="JHipster is an open source, widely used application generator, made for modern full-stack developers." />
```
Comment (jdubois) [inline _layouts/default.html:12]: "open source" or "Open Source"?

File: index.html (hunk 1 above)
Comment (mraible) [reply index.html:30]: I've learned through all my editors when writing that it's better to not capitalize words. It's kinda like internet, which is [not supposed to be capitalized any more](https://www.wired.com/2015/10/should-you-be-capitalizing-the-word-internet/).

Comment (jdubois) [reply index.html:30]: Oh I've learned something today, thank you!

File: _layouts/default.html (hunk 3 above)
Comment (mraible) [reply _layouts/default.html:7]: Bruce Perens [agrees with you](https://perens.com/2017/09/26/on-usage-of-the-phrase-open-source/). According to [this reference](https://english.stackexchange.com/questions/157178/capitals-or-not-for-open-source), it's correct as I have it.

Comment (jdubois) [reply _layouts/default.html:7]: If we write "web", then let's also write "open source"

File: index.html [hunk 5]
```diff
@@ -482,55 +485,46 @@ <h2 class="heading">What is JHipster?</h2>
...
-                        Our goal is to generate for you a complete and modern Web app or microservice architecture, unifying:
+                        Our goal is to generate a complete and modern web app or microservice architecture, unifying:
                     </p>
                     <ul>
-                        <li>A high-performance and robust Java stack on the server side with Spring Boot</li>
-                        <li>A sleek, modern, mobile-first front-end with Angular, React and Bootstrap</li>
-                        <li>A robust microservice architecture with the JHipster Registry, Netflix OSS, the Elastic Stack and Docker</li>
-                        <li>A powerful workflow to build your application with Yeoman, Webpack and Maven/Gradle</li>
+                        <li>A high-performance and robust server-side stack with excellent test coverage</li>
+                        <li>A sleek, modern, mobile-first UI with Angular, React, or Vue + Bootstrap for CSS</li>
```
Comment (sendilkumarn) [inline index.html:501]: nit: ```suggestion
                        <li>A sleek, modern, mobile-first UI with Angular, React, or Vue + Bootstrap CSS framework</li>
```

Comment (mraible) [reply index.html:501]: I'm not sure we should add framework since technically Angular, React, and Vue are all frameworks.

**Comment by deepu105** [comment]
Cool

**Comment by mraible** [comment]
@jdubois It might be a good idea to update https://twitter.com/jhipster to have similar changes.

**Pull request merged**